docs/additional-functionality/rapids-udfs.md

```diff
@@ -134,31 +134,35 @@ implements a Hive simple UDF using
 [native code](../../udf-examples/src/main/cpp/src) to count words in strings
 
 
-## GPU Scheduling For Pandas UDF
+## GPU Support for Pandas UDF
 
 ---
 **NOTE**
 
-The _GPU Scheduling for Pandas UDF_ is an experimental feature, and may change at any point it time.
+The GPU support for Pandas UDF is an experimental feature, and may change at any point it time.
 
 ---
 
-_GPU Scheduling for Pandas UDF_ is built on Apache Spark's [Pandas UDF(user defined
-function)](https://spark.apache.org/docs/3.0.0/sql-pyspark-pandas-with-arrow.html#pandas-udfs-aka-vectorized-udfs),
-and has two components:
+GPU support for Pandas UDF is built on Apache Spark's [Pandas UDF(user defined
+function)](https://spark.apache.org/docs/latest/api/python/user_guide/arrow_pandas.html#pandas-udfs-a-k-a-vectorized-udfs),
+and has two features:
 
-- **Share GPU with JVM**: Let the Python process share JVM GPU. The Python process could run on the
-  same GPU with JVM.
+- **GPU Assignment(Scheduling) in Python Process**: Let the Python process share the same GPU with
+Spark executor JVM. Without this feature, in a non-isolated environment, some use cases with
+PandasUDF (a.k.a an `independent` process) can try to use GPUs other than the one we want it to
+run on. For example, the user could launch a TensorFlow session inside Pandas UDF and the machine
+contains 8 GPUs. Without this GPU sharing feature, TensorFlow will automatically use all 8 GPUs
+which will conflict with existing Spark executor JVM processes.
 
-- **Increase Speed**: Make the data transport faster between JVM process and Python process.
+- **Increase Speed**: Speeds up data transfer between JVM process and Python process.
 
 
 
-To enable _GPU Scheduling for Pandas UDF_, you need to configure your spark job with extra settings.
+To enable GPU support for Pandas UDF, you need to configure your Spark job with extra settings.
 
-1. Make sure GPU exclusive mode is disabled. Note that this will not work if you are using exclusive
-   mode to assign GPUs under spark.
-2. Currently the python files are packed into the spark rapids plugin jar.
+1. Make sure GPU `exclusive` mode is disabled. Note that this will not work if you are using exclusive
```

Comment:
For On-prem YARN instructions, we add a line "refer to nvidia-smi documentation". 

Do we need to add the link or sample like this? 
nvidia-smi -i 0 -c EXCLUSIVE_PROCESS # Set GPU 0 to exclusive mode, run as root.

Also, if this mandatory for YARN, should we say that more explicitly?

Reply:
Thanks for review!
Added a line to show the command to set GPU to Default mode.
I think no need to say if it's for yarn or for other environment. The _mode_ here is only for GPU hardware. If the GPU is in Default mode, it will always be default mode no matter on Yarn or on Standalone or Local.